src/utils/evm/testEVMToEVMRoutes.ts

```diff
@@ -0,0 +1,231 @@
+
+import { EVMAssetTransfer, EVMGenericMessageTransfer, Environment, EthereumConfig, EvmResource, Resource, ResourceType } from '@buildwithsygma/sygma-sdk-core'
+import { BigNumber, Wallet, providers, utils } from 'ethers'
+import { RpcEndpoints } from '../../types'
+
+import { Storage__factory } from '../Contracts'
+import { LoggingData, fetchTokenAmount, sleep, waitUntilBridgedFungibleEvm } from '../helpers'
+import { print } from 'gluegun'
+
+const executionContractAddresses = {
+  EXECUTE_CONTRACT_ADDRESS_1:
+    '0xdFA5621F95675D37248bAc9e536Aab4D86766663',
+  EXECUTE_CONTRACT_ADDRESS_2:
+    '0x0e963aEe445EDC19034e9938570E5E7BE4Ee19Cd'
+}
+const DEPOSIT_AMOUNT = '500000000000'
```

Comment:
Can we move all of these constants to something like this:
```
const  DEPOSIT_AMOUNT  = process.env.DEPOSIT_AMOUNT || '500000000000'
```
So, it is possible to set this up using ENV variables. Also, add these values to the README file for this specific script.